.github/workflows/image-build.yaml

```diff
@@ -64,19 +129,21 @@ jobs:
               # set latest tag for default branch
               type=raw,value=latest,enable={{is_default_branch}}
 
-    name: Build Image (${{ matrix.image.image }})
+    name: Build Bundle Image (${{ matrix.image.image }})
     steps:
-      - uses: actions/checkout@v4
+      - name: Checkout bpfman
+        uses: actions/checkout@v4
 
-      - uses: actions/setup-go@v5
+      - name: Install Golang
```

Comment:
I don't think we'll even need this for just `make bundle-build`

Reply:
Removed